core/src/avm2/globals/flash/display3D/context_3d.rs

```diff
@@ -612,15 +612,6 @@ pub fn set_render_to_texture<'gc>(
         );
     }
 
-    if surface_selector != 0 {
```

Comment:
Do we want the same validation as FP has?
> ArgumentError - for a mismatched surfaceSelector parameter. The value must be 0 for 2D textures and 0..5 for cube maps

Reply:
I hesitated to merge myself because I wasn't sure how strongly you wanted this to be addressed.